apis/connectivity/v1alpha1/types.go

```diff
@@ -14,6 +14,7 @@ const (
 	ImportedLabel           = "connectivity.tanzu.vmware.com/imported"
 	GlobalVIPAnnotation     = "connectivity.tanzu.vmware.com/vip"
 	FQDNAnnotation          = "connectivity.tanzu.vmware.com/fqdn"
+	ARecordAnnotation       = "capi-dns/a-record"
```

Comment:
Worth getting some feedback on whether or not we want to have `capi-dns` be anything else.

Reply:
Yeah.  btw, I've updated my draft design doc to have it as `capi-dns/hostname` now, with the idea that the DNS server should be able to decide what kind of records to serve for the hostname, based on `EndpointSlice.spec.addressType`

Reply:
Perhaps it makes sense to document that the product only supports ipv4 in its current state?

Reply:
That is in the Readme, unless you are thinking we document something else?